core/src/main/java/org/infinispan/configuration/global/GlobalTracingConfigurationBuilder.java

```diff
@@ -0,0 +1,92 @@
+package org.infinispan.configuration.global;
+
+import static org.infinispan.configuration.global.GlobalTracingConfiguration.COLLECTOR_ENDPOINT;
+import static org.infinispan.configuration.global.GlobalTracingConfiguration.ENABLED;
+import static org.infinispan.configuration.global.GlobalTracingConfiguration.EXPORTER_PROTOCOL;
+import static org.infinispan.configuration.global.GlobalTracingConfiguration.SECURITY;
+import static org.infinispan.configuration.global.GlobalTracingConfiguration.SERVICE_NAME;
+
+import org.infinispan.commons.configuration.Builder;
+import org.infinispan.commons.configuration.Combine;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+
+public class GlobalTracingConfigurationBuilder extends AbstractGlobalConfigurationBuilder implements Builder<GlobalTracingConfiguration> {
+
+   private final AttributeSet attributes;
+
+   public GlobalTracingConfigurationBuilder(GlobalConfigurationBuilder globalConfig) {
+      super(globalConfig);
+      this.attributes = GlobalTracingConfiguration.attributeDefinitionSet();
+   }
+
+   @Override
+   public AttributeSet attributes() {
+      return attributes;
+   }
+
+   public String collectorEndpoint() {
+      return attributes.attribute(COLLECTOR_ENDPOINT).get();
+   }
+
+   public GlobalTracingConfigurationBuilder collectorEndpoint(String collectorEndpoint) {
+      attributes.attribute(COLLECTOR_ENDPOINT).set(collectorEndpoint);
+      return this;
+   }
+
+   public boolean enabled() {
+      return collectorEndpoint() != null && attributes.attribute(ENABLED).get();
+   }
+
+   public GlobalTracingConfigurationBuilder enable() {
+      attributes.attribute(ENABLED).set(true);
+      return this;
+   }
+
+   public GlobalTracingConfigurationBuilder disable() {
+      attributes.attribute(ENABLED).set(false);
+      return this;
+   }
+
+   public TracingExporterProtocol exporterProtocol() {
+      return attributes.attribute(EXPORTER_PROTOCOL).get();
+   }
+
+   public GlobalTracingConfigurationBuilder exporterProtocol(TracingExporterProtocol exporterProtocol) {
+      attributes.attribute(EXPORTER_PROTOCOL).set(exporterProtocol);
+      return this;
+   }
+
+   public String serviceName() {
+      return attributes.attribute(SERVICE_NAME).get();
+   }
+
+   public GlobalTracingConfigurationBuilder serviceName(String serviceName) {
+      attributes.attribute(SERVICE_NAME).set(serviceName);
+      return this;
+   }
+
+   public boolean traceSecurity() {
+      return enabled() && attributes.attribute(ENABLED).get();
```

Comment:
getter methods like this aren't required and this implementation is incorrect.

Reply:
fixed thanks